click/README.md

```diff
@@ -0,0 +1,269 @@
+# Click on Unikraft
+
+This application starts a [Click](https://github.com/sysml/clickos) program with Unikraft.
+Follow the instructions below to set up, configure, build and run Click.
+
+## Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.
+Note that you still need to install the [requirements](#requirements).
+
+For building and running everything for Qemu in `x86_64`, follow the steps below:
+
+```console
+./setup.sh
+UK_DEFCONFIG=$(pwd)/.config.click-qemu-x86_64 make defconfig
```

Comment:
This should be replaced with a `wget` from the scripts branch, like in the other applications